pyproject.toml

```diff
@@ -31,6 +31,7 @@ Issues = "https://github.com/RealOrangeOne/seahash-py/issues"
 
 [project.optional-dependencies]
 dev = [
-  "pytest==7.1.2",
-  "black==22.6.0"
+    "maturin",
```

Comment:
`maturin` shouldn't need to be pinned explicitly, it's part of the install process.

Similarly, we should pin the dependencies here.

Reply:
Done